hera_qm/xrfi.py

```diff
@@ -225,6 +199,3 @@ def xrfi(d, f=None, K=8, sig_init=6, sig_adj=2):
     return f
 
 # XXX split off median filter as one type of flagger
```

Comment:
Is this supposed to be a TODO? If so, let's make it and issue and delete the comment in the code.